fhservices.cfg.j2

```diff
@@ -139,15 +144,24 @@ define service {
 }
 
 define service {
-       service_description Database::MongoDB::Health
-       check_command check_mongodb!mongodb,mongodb-service
+       service_description mongodb:health
+       check_command check_mongodb!mongodb,mongodb-service,mongodb-1
```

Comment:
Is this supposed to fix the currently failing health check for mongodb on the core? I'm still seeing an error:

```
CRIT: There are 0 nodes claiming to be primary members of the replica set
```
Created a ticket about this issue also https://issues.jboss.org/browse/RHMAP-9900